docker/manager.go

```diff
@@ -31,10 +31,10 @@ import (
 
 // Manager manages all docker operations, like build, push, etc.
 type Manager struct {
-	client     *docker_client.Client
-	registry   string
-	authConfig *AuthConfig
-	endPoint   string
+	Client     *docker_client.Client
+	Registry   string
+	AuthConfig *AuthConfig
+	EndPoint   string
```

Comment:
Would exposed the vars violate the golang's coding style?  if yes,  i think can delete the functions, as GetDocker***